scripts-vagrant_provision/diagnostics.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+ALMOST_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+DIR="${ALMOST_DIR}/.."
+export $(grep -v '^#' "${DIR}/.env" | xargs -d '\n')
+
+summary_images ()
+{
+    service_string=$(printenv ENABLED_SERVICES)
+    own_services=( "frontend" "core" "events" "statutory" )
```

Comment:
This should include the others as well, right? Discounts, network, gsuite-wrapper, summeruniversity, mailer. Possibly knowledge but we'll likely remove that soon